Patches justfile to make correct symlinks like dosym would

diff --git a/justfile b/justfile
index a7d196f..b57c764 100644
--- a/justfile
+++ b/justfile
@@ -15,7 +15,7 @@ prefixdir := prefix + '/bin'
 bindir := rootdir + prefixdir
 default-schema-target := sharedir / 'cosmic'
 
-cosmic-applets-bin := prefixdir / 'cosmic-applets'
+cosmic-applets-bin := '/usr/bin/cosmic-applets'
 
 default: build-release

